stable/odoo/README.md

```diff
@@ -45,31 +45,55 @@ The command removes all the Kubernetes components associated with the chart and
+| `ingress.hosts[0].name`               | Hostname to your Odoo installation                          | `odoo.local`                                   |
+| `ingress.hosts[0].path`               | Path within the url structure                               | `/`                                            |
+| `ingress.hosts[0].tls`                | Utilize TLS backend in ingress                              | `false`                                        |
+| `ingress.hosts[0].tlsSecret`          | TLS Secret (certificates)                                   | `odoo.local-tls-secret`                        |
+| `ingress.hosts[0].annotations`        | Annotations for this host's ingress record                  | `[]`                                           |
+| `ingress.secrets[0].name`             | TLS Secret Name                                             | `nil`                                          |
+| `ingress.secrets[0].certificate`      | TLS Secret Certificate                                      | `nil`                                          |
+| `ingress.secrets[0].key`              | TLS Secret Key                                              | `nil`                                          |
+| `resources`                           | CPU/Memory resource requests/limits                         | Memory: `512Mi`, CPU: `300m`                   |
+| `persistence.storageClass`            | PVC Storage Class                                           | `nil` (uses alpha storage class annotation)    |
+| `persistence.accessMode`              | PVC Access Mode                                             | `ReadWriteOnce`                                |
+| `persistence.size`                    | PVC Storage Request                                         | `8Gi`                                          |
+| `postgresql.postgresqlPassword`       | PostgreSQL password                                         | `nil`                                          |
+| `postgresql.persistence.enabled`      | Enable PostgreSQL persistence using PVC                     | `true`                                         |
+| `postgresql.persistence.storageClass` | PVC Storage Class for PostgreSQL volume                     | `nil` (uses alpha storage class annotation)    |
+| `postgresql.persistence.accessMode`   | PVC Access Mode for PostgreSQL volume                       | `ReadWriteOnce`                                |
+| `postgresql.persistence.size`         | PVC Storage Request for PostgreSQL volume                   | `8Gi`                                          |
+| `livenessProbe.enabled`               | Enable/disable the liveness probe                           | `true`                                         |
+| `livenessProbe.initialDelaySeconds`   | Delay before liveness probe is initiated                    | 120                                            |
```

Comment:
this does not match the value in values.yaml. There are other wrongs too

Reply:
👍